docs/source/installing-brave.rst

```diff
@@ -243,3 +244,18 @@ CentOS/RHEL
     sudo yum install brave-browser-nightly
 
 The key you're importing should have fingerprint ``9228 DBCE 20DD E5EC 4648  8DE9 0B31 DBA0 6A8A 26F9``.
+
+
+Unofficial packages
+============================
+
+.. highlight:: console
+
+Solus 
+-----------
+::
+
+    sudo eopkg it brave
```

Comment:
This will be only for release channel I suppose?

Reply:
Yeah that is correct

Reply:
May be good to make it `Unofficial packages (Release Channel only)` in a follow up PR.

Reply:
I will see what I can do and follow up with a new PR in a few days 👍